a10_octavia/controller/worker/tasks/health_monitor_tasks.py

```diff
@@ -75,13 +75,18 @@ def execute(self, listeners, health_mon, vthunder, flavor=None):
                           "A health monitor of type {} is not supported "
                           "by A10 provider").format(health_mon.id, health_mon.type))
 
+        override_port = CONF.health_monitor.use_override_port
+        port = None
+        if override_port:
+            port = listeners[0].protocol_port
```

Comment:
This should just be the default. Health monitor will use the member port if it is defined instead of its own

Reply:
Fixed